cmd/backup-manager/app/import/restore.go

```diff
@@ -58,14 +58,14 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
 		return fmt.Errorf("dir %s does not exist or is not a dir", restorePath)
 	}
 	args := []string{
-		"--status-addr=0.0.0.0:8289",
-		"--backend=tidb",
-		"--server-mode=false",
-		"–-log-file=",
-		fmt.Sprintf("--tidb-user=%s", ro.User),
-		fmt.Sprintf("--tidb-password=%s", ro.Password),
-		fmt.Sprintf("--tidb-host=%s", ro.Host),
-		fmt.Sprintf("--d=%s", restorePath),
+		"-status-addr=0.0.0.0:8289",
```

Comment:
The options in the tidb-lightning chart are also with `--` prefix, could you please double confirm the format? 
https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-lightning/templates/scripts/_start_lightning.sh.tpl#L28